x-pack/plugins/observability_solution/infra/public/plugin.ts

```diff
@@ -440,6 +435,19 @@ const getLogsNavigationEntries = ({
   return entries;
 };
 
+const getLogsExplorerAccessibility$ = (application: CoreStart['application']) => {
+  const { capabilities, applications$ } = application;
+  return applications$.pipe(
+    map(
+      (apps) =>
+        (apps.get(OBSERVABILITY_LOGS_EXPLORER_APP_ID)?.status ?? AppStatus.inaccessible) ===
+          AppStatus.accessible &&
+        capabilities.discover?.show &&
+        capabilities.fleet?.read
+    )
+  );
```

Comment:
```suggestion
    ),
    distinctUntilChanged()
  );
```
```
// `import { map, distinctUntilChanged } from 'rxjs';`
```

to avoid duplicate values. Otherwise the subscription may be called multiple of times resulting in the following:

<img width="1097" alt="198262-multi-nav-02" src="https://github.com/user-attachments/assets/5ca0c48a-4d8a-468b-8664-3086d88b7a1a">

The above is observed on `8.x` branch and wasn't happening on `main`, but would be ideal to adjust everywhere.

Also see if we need `capabilities.discover?.show && capabilities.fleet?.read` explicitly here as the following already includes both.
https://github.com/elastic/kibana/blob/3feea7bcfc398cb754efe1504928d7e62e3a4e55/x-pack/plugins/observability_solution/observability_logs_explorer/public/plugin.ts#L150